packages/accordions/examples/basic.md

```diff
@@ -0,0 +1,249 @@
+### Steppers
+
+The `Stepper` component accepts an `activeIndex` prop and determines the current step and computes
+the completed steps. The `activeIndex` can be used to help communicate to assistive devices in
+which step the user is currently on using
+[ARIA live regions](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions).
+
+The default layout of the `Stepper` is vertical, and renders step content using `StepContent`.
```

Comment:
That's a fair point. My goal was to ensure consumers know that horizontal Stepper requires content to live outside it. What do you think if the entire paragraph was reduced to the following sentence?

> For a horizontal `Stepper`, consumers should render step content outside of the `Stepper` component instead of using `Stepper.Content`.

Reply:
Looks good!